content/sensu-go/6.2/operations/deploy-sensu/secure-sensu.md

```diff
@@ -203,6 +203,344 @@ However, deployments can also use the same certificates and keys for etcd peer a
+        "filters": [
+          "entity.system.os == 'linux'",
+          "entity.system.arch == 'armv7'"
+        ],
+        "headers": null,
+        "sha512": "f7ad60cfc4ba12ccfe623e2d86727a544b35c2fb165f6c9acae432f132762022ba542cf26d92e15ac445cc5d241feb6055c17286cb2f5a1e5c6be0723a003e91",
+        "url": "https://assets.bonsai.sensu.io/7d72df2682c0134068abd21a65778c81e61d7afe/system-profile-linux_0.10.1_linux_armv7.tar.gz"
+      },
+      {
+        "filters": [
+          "entity.system.os == 'linux'",
+          "entity.system.arch == 'arm64'"
+        ],
+        "headers": null,
+        "sha512": "689def9af9116cdeaf8bca319b59012a83073bc05587e3e974aa5b9a5cd3a40fbab7ce386737d1d18c62d9d06b7393d3cdc26e35c8c697920cc1a9a9acb94ad9",
+        "url": "https://assets.bonsai.sensu.io/7d72df2682c0134068abd21a65778c81e61d7afe/system-profile-linux_0.10.1_linux_arm64.tar.gz"
+      },
+      {
+        "filters": [
+          "entity.system.os == 'linux'",
+          "entity.system.arch == '386'"
+        ],
+        "headers": null,
+        "sha512": "7c5bf18b52f6c6b9d55f1fe882c4341949730d25abf31fc472d6779948d31bc3606887ceb7d160d706acd5abd660111e98ba0220815a87d1a5c351dd95b2eaf3",
+        "url": "https://assets.bonsai.sensu.io/7d72df2682c0134068abd21a65778c81e61d7afe/system-profile-linux_0.10.1_linux_386.tar.gz"
+      },
+      {
+        "filters": [
+          "entity.system.os == 'linux'",
+          "entity.system.arch == 'amd64'"
+        ],
+        "headers": null,
+        "sha512": "29a6e1725f37e533cd9f6a8a17b40ef757ba5ea5dc27701c19a7fb9ac9c2bdf9f684d758d4440f278bb7fc669e52bf02efb3fc6758731f7f3e10617a2fe026cf",
+        "url": "https://assets.bonsai.sensu.io/7d72df2682c0134068abd21a65778c81e61d7afe/system-profile-linux_0.10.1_linux_amd64.tar.gz"
+      }
+    ],
+    "filters": null,
+    "headers": null
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+### Target a specific dynamic runtime asset directory
+
+To improve security, configure Sensu access to use `sudo -E` only for specific assets installed under `var/cache/sensu/sensu-agent/`, 
+In your `/etc/sudoers.d/` file, in place of `**`, list the sha512 sum for the specific build of the dynamic runtime asset you are using.
+This effectively whitelists the asset paths you want sudo to be able to use.
+You can find the sha512 sum for the build you want to whitelist in the asset definition.
+
+For example, the following rules would allow `sudo -E` access for only the [Linux amd64 build of the System Profile Linux][13] dynamic runtime asset:
+
+{{< code shell >}}
+sensu   ALL=(ALL)       NOPASSWD:SETENV: /var/cache/sensu/sensu-agent/29a6e1725f37e533cd9f6a8a17b40ef757ba5ea5dc27701c19a7fb9ac9c2bdf9f684d758d4440f278bb7fc669e52bf02efb3fc6758731f7f3e10617a2fe026cf/bin/*.*
+sensu   ALL=(ALL)       NOPASSWD:SETENV: /var/cache/sensu/sensu-agent/29a6e1725f37e533cd9f6a8a17b40ef757ba5ea5dc27701c19a7fb9ac9c2bdf9f684d758d4440f278bb7fc669e52bf02efb3fc6758731f7f3e10617a2fe026cf/bin/*
+{{< /code >}}
```

Comment:
We should axe this whole section, do not offer globs as a pattern then there is no need to ask people to do security measures they wont do given the alternative.